packages/expo-av/CHANGELOG.md

```diff
@@ -7,12 +7,13 @@
 ### 🎉 New features
 
 - [av] Delete `prop-types` in favor of TypeScript. ([#8679](https://github.com/expo/expo/pull/8679) by [@EvanBacon](https://github.com/EvanBacon))
+- [av] Directly import `createElement` from `react-native-web` for RNW v12 support. ([#8773](https://github.com/expo/expo/pull/8773) by [@EvanBacon](https://github.com/EvanBacon))
```

Comment:
I think we don't need to prefix entries by the package name like `[av]` 🤔 That's kinda obvious they refer to that package. Also, the script that merges changelog into SDK changelog groups them by package like here: https://github.com/expo/expo/blob/master/CHANGELOG.md#-bug-fixes-1

```suggestion
- Directly import `createElement` from `react-native-web` for RNW v12 support. ([#8773](https://github.com/expo/expo/pull/8773) by [@EvanBacon](https://github.com/EvanBacon))
```

Reply:
the changelog danger script added the prefix